### DISCUSSION TITLE
allow DHCP IP address to be published via websend (no 0.0.0.0 prefix)

### DISCUSSION THREAD

**Discussion by Kugeleis** [arendst/Tasmota#12567] [Q&A] [answered]
The ipaddress1 is in the form 0.0.0.0 (192.168.1.2). This is not useful to publish it via websend. Could tasmota expose a pure version of its actual IP address as 192.168.1.2?
**Use case:** Two tasmota devices communicate via websend. One device travels between networks and should publish it's actual IP address via websend. I have this working but the string `var1 = 0.0.0.0 (192.168.1.2)` is not useful for the other device when it wants to `websend [%var1%] payload` since `websend [0.0.0.0 (192.168.1.2)] payload` fails.

**Comment by barbudor** [comment, marked as answer]
Can't you use `on Info2#IpAddress do ....` that is published right after a boot ?

**Reply by barbudor** [reply]
By the way, your use case would much easier when using MQTT

**Comment by Kugeleis** [comment]
Thanks, Info2#IpAddress does the job. This should be told in the docs more explicitly.
Actually, I have MQTT running but less nodes less trouble.
